lib/create-plugin.js

```diff
@@ -121,6 +123,15 @@ function createPlugin(instrumentationApi) {
           // default. only used if failed to parse
           let transactionName = '*'
           let segmentName = UNKNOWN_OPERATION
+          let query = responseContext.request.query
```

Comment:
Might want to do this work only if agent config attributes includes 'graphql.operation.query'.  Thoughts?